modules/doc/content/application_development/profiling.md

```diff
@@ -47,6 +47,22 @@ This will use the filename base you pass and append a suffix of the form
 rank/process. The above example would generate files `run1_0.prof`,
 `run1_1.prof`, `run1_2.prof`, ..., `run1_31.prof`.
 
+### Heap Profiling
+
+Similarly, you can do a heap profiling like this:
+
+```
+HEAP_PROFILE_INUSE_INTERVAL=104857600 MOOSE_HEAP_BASE=run1_ mpiexec -n 32 ./your-app_oprof -i input_file.i
```

Comment:
Isn't this the default value? Does anything happen if you don't specify this environment value? I remember when I was trying to do this, I would not get any memory profiles dumped ever, but I wasn't specifying any environment variable values like `HEAP_PROFILE_INUSE_INTERVAL`

Reply:
The default is nothing. Right, you do not get anything if you do not specify `HEAP_PROFILE_INUSE_INTERVAL` or other variables